src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -63,50 +63,50 @@ _* There are a few Nerdlets that basic users are allowed to use. See [Basic user
 
 ## Nerdpack permissions: Original user model
 
-Learn the differences between how basic users, full users, and admins can use and manage Nerdpacks in the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/).
+Learn the differences between how basic users, full platform users, and admins can use and manage Nerdpacks in the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/).
 
 ### Basic users
 
-If you're a basic user in the original user model, you're limited in your Nerdpack capabilities. You can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use your Nerdpacks, an admin must upgrade you to a full user.
+If you're a basic user in the original user model, you're limited in your Nerdpack capabilities. You can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use your Nerdpacks, an admin must upgrade you to a full platform user/Core user.
```

Comment:
Re-reading this, I think we should leave this as "full user". There weren't core users in the original model and I don't think we called full users "full platform users" back then.